solutions/standard/variables.tf

```diff
@@ -209,6 +209,65 @@ variable "auto_scaling" {
   default     = null
 }
 
+##############################################################################
+## Secrets Manager Service Credentials
+##############################################################################
+
+variable "skip_rd_sm_auth_policy" {
+  type        = bool
+  default     = false
+  description = "Whether an IAM authorization policy is created for Secrets Manager instance to create a service credential secrets for Databases for Redis. If set to false, the Secrets Manager instance passed by the user is granted the Key Manager access to the Redis instance created by the Deployable Architecture. Set to `true` to use an existing policy. The value of this is ignored if any value for 'existing_secrets_manager_instance_crn' is not passed."
+}
```

Comment:
Could this be moved to the bottom for easier validation against elasticsearch? Thanks.